web/src/OverviewTableColumns.tsx

```diff
@@ -533,7 +617,6 @@ const DEFAULT_COLUMNS: Column<RowValues>[] = [
   {
     Header: () => <TableHeaderStarIcon title="Starred" />,
     id: "starred",
-    accessor: "name", // Note: this accessor is meaningless but required when `Header` returns JSX. The starred column gets its data directly from the StarredResources context and sort on this column is disabled.
```

Comment:
huh! did this comment turn out to be wrong, or was there some other change I missed?

(if only I'd also commented what I observed to make me think it was required)

Reply:
that's a great question! what i remember (when adding that comment) is that there was a typescript compilation error, saying that `accessor` was required, even though [react-table's docs](https://react-table.tanstack.com/docs/api/useTable#column-options) say that you can omit it when you have an id. (i've reread these docs a couple times and my take-away is a little bit different each time. 😅) 

i didn't get the same ts complaint this time around, which is why i removed the field. because both the starred and selection columns get data from context, rather than table data, i think it's okay to omit the `accessor` property. 

(however, this is making me consider that we should be mapping the context data to the table row data, instead of grabbing the data from whatever column component, and then that data would be sortable, etc. definitely a thought for another pr though!)